src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContractSerializer.cs

```diff
@@ -66,6 +66,8 @@ public DataContractSerializer(Type type, XmlDictionaryString rootName, XmlDictio
 
 #if NET_NATIVE
         public DataContractSerializer(Type type, IEnumerable<Type> knownTypes, int maxItemsInObjectGraph, bool ignoreExtensionDataObject, bool preserveObjectReferences)
+#elif MERGE_DCJS
+        internal DataContractSerializer(Type type, IEnumerable<Type> knownTypes, int maxItemsInObjectGraph, bool ignoreExtensionDataObject, bool preserveObjectReferences)
```

Comment:
Yes, this is not right. My intention is that this constructor only exists in NetNative or with the new DCJS with different visibility.